lidarrmetadata/provider.py

```diff
@@ -759,31 +564,37 @@ def get_album_media(self, album_id):
                  'Position': result['medium_position']}
                 for result in results]
 
-    def get_album_tracks(self, album_id):
-        results = self.query_from_file('track_album_mbid.sql',
-                                       [album_id])
+    def get_release_group_tracks(self, rgid):
+        results = self.query_from_file('track_release_group.sql', [rgid])
 
         return [{'Id': result['gid'],
+                 'ReleaseId': result['release_id'],
+                 'ArtistId': result['artist_id'],
                  'TrackName': result['name'],
                  'DurationMs': result['length'],
                  'MediumNumber': result['medium_position'],
                  'TrackNumber': result['number'],
                  'TrackPosition': result['position']}
                 for result in results]
 
-    def get_albums_by_artist(self, artist_id):
-        results = self.query_from_file('album_search_artist_mbid.sql',
+    def get_release_groups_by_artist(self, artist_id):
+        results = self.query_from_file('release_group_search_artist_mbid.sql',
                                        [artist_id])
 
         return [{'Id': result['gid'],
+                 'ArtistId': artist_id,
                  'Disambiguation': result['comment'],
                  'Title': result['album'],
                  'Type': result['primary_type'],
                  'SecondaryTypes': result['secondary_types'],
                  'ReleaseStatuses': result['release_statuses'],
                  'ReleaseDate': datetime.datetime(result['year'] or 1,
                                                   result['month'] or 1,
-                                                  result['day'] or 1)}
+                                                  result['day'] or 1),
+                 'Rating': {
+                     'Count': result['rating_count'],
+                     'Value': (result['rating'] or 0) / 10 or None
```

Comment:
I was thinking that there was some reason that I wrote it like this for the artist rating, but `0 / 10 or None` will return `None` in the case of the rating being `0`. Maybe something like `result['rating']  / 10 if result['rating'] is not None else None`?